getting_started/scripting/gdscript/gdscript_basics.rst

```diff
@@ -1256,6 +1256,9 @@ placed at the top of the file::
     func _ready():
         print("Hello")
 
+
+See :ref:`doc_running_code_in_the_editor` for more info.
```

Comment:
This sounds a bit more professional to me:

```suggestion
See :ref:`doc_running_code_in_the_editor` for more information.
```

Other than that, it looks fine to me.

Reply:
Done, thanks!